VersionControlUI/src/DiffManager.h

```diff
@@ -181,23 +204,30 @@ class VERSIONCONTROLUI_API DiffManager
 		 * Check if change associated with id should be visualized (e.g. if the change is part of a name change, does it
 		 * satisfy the visualization constraint).
 		 */
-		bool shouldShowChange(Model::NodeIdType id);
+		static bool shouldShowChange(Model::NodeIdType id);
 
 		QString project_;
 		QList<Model::SymbolMatcher> contextUnitMatcherPriorityList_;
 
 		// if set specifies which node we are interested in, used for history
 		Model::NodeIdType targetNodeId_;
 
-		QHash<QString, QPair<QString, Model::NodeIdType>> nameChanges_;
-		QHash<Model::NodeIdType, bool> nameChangesIdsIsNameText_;
-		NameChangeVisualizations nameChangeVisualization_{Summary};
+		QHash<QString, QPair<QString, Model::NodeIdType>> nameChangeInformation_;
+		static QHash<Model::NodeIdType, bool> nameChangesIdsIsNameText_;
```

Comment:
Uf, there's a ton of these statics related to name changes. Could you at least put some comments, to explain in a sentence or two what each one is for? Is it possible to perhaps combine some of these (maybe by making an additional class/struct)? Would such a combination be a better design?